sdk/template-LLC/Azure.Template.LLC/src/swagger/swagger.json

```diff
@@ -1,123 +1,121 @@
-  "produces": ["application/json"],
-  "consumes": ["application/json"],
+  "schemes": [
+    "http"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "consumes": [
+    "application/json"
+  ],
   "paths": {
-    "/test1": {
-      "get": {
-        "operationId": "NoRequestBodyResponseBody",
-        "description": "No RequestBody and ResponseBody",
+    "/providers/Microsoft.Template/resource/{resourceId}": {
+      "put": {
+        "operationId": "Create",
+        "description": "Create or update resource.",
         "parameters": [
           {
-            "in": "query",
-            "name": "$top",
-            "description": "Query parameter top",
-            "type": "integer",
-            "required": false
-          },
-          {
-            "in": "query",
-            "name": "id",
-            "description": "Query parameter Id",
-            "type": "integer",
+            "name": "resourceId",
+            "in": "path",
             "required": true,
-            "x-nullable": true
-          },
-          {
-            "in": "query",
-            "name": "skip",
-            "description": "Query parameter skip",
-            "type": "integer",
-            "required": true,
-            "x-ms-client-default": 12
-          },
-          {
-            "in": "query",
-            "name": "status",
-            "description": "Query parameter status",
             "type": "string",
-            "x-ms-client-default": "start"
+            "description": "The id of the resource."
           }
         ],
         "responses": {
           "200": {
-            "description": "String Response",
+            "description": "OK",
             "schema": {
-              "type": "string"
+              "$ref": "#/definitions/Resource"
             }
           }
         }
       },
-      "put": {
-        "operationId": "RequestBodyResponseBody",
-        "description": "RequestBody and ResponseBody",
+      "delete": {
+        "operationId": "Delete",
+        "description": "Delete resource.",
         "parameters": [
           {
-            "name": "model",
-            "description": "body",
-            "in": "body",
-            "schema": {
-              "$ref": "#/definitions/TestModel"
-            }
+            "name": "resourceId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The id of the resource."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          }
+        }
+      },
+      "get": {
+        "operationId": "Get",
+        "description": "Retrieves information about the resource.",
+        "parameters": [
+          {
+            "name": "resourceId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The id of the resource."
           }
         ],
         "responses": {
           "200": {
```

Comment:
DELETE should return a 204: https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#http-request--response-pattern